README.md

```diff
@@ -139,16 +139,14 @@ technical reference for APIs and other aspects of beaker-pe. They describe
 how it works and how to use it but assume that you have a basic understanding
 of key concepts.
 
+Add beaker's [contributing guide](https://github.com/puppetlabs/puppet/blob/master/CONTRIBUTING.md) in your
+documentation. If the library is developed and supported by puppet it is important to follow
+that guide for future changes to your library.
```

Comment:
@rishijavia it's not really true that our recommendation is to add beaker's contributing guide to the project. That makes it sound like we'd like them to copy our file, unlike what we talked about yesterday. We'd rather have them have a contributing section to their README that points to beaker's contributing guide as a starting point.

I think the best route here would be to have an example of the text for them to use as their contributing section, like how we have an example of the Rubydocs section of the doc. This way, they can even easier use our default and get decent contributing standards without much effort at all.